src/main/java/monitoringsystemturbo/presenter/MainPresenter.java

```diff
@@ -305,4 +308,18 @@ public void setApplicationListController(ApplicationListController applicationLi
     public void setExportController(ExportController exportController) {
         this.exportController = exportController;
     }
+
+
+    public void setMotivesController(MotivesController motivesController) {
+        this.motivesController = motivesController;
+    }
+
+    @FXML
+    public void onMotives() {
+        try {
+            motivesController.showMotivesView();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Are you sure we don't want to display some message to a user, but just print a stack trace? If it is an important error which could cause an unexpected app behavior consider notifying user about that